style.css

```diff
@@ -316,3 +304,310 @@ footer {
   justify-content: space-evenly;
   margin-top: 60px;
 }
+
+.nav-list {
+  display: none;
+}
+
+@media screen and (min-width: 1024px) {
+  body {
+    background-image: none;
+    max-width: 1400px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  /* Hero-section */
+  .header {
+    justify-content: center;
+    border-bottom: 1px solid #dddfe6;
+    padding-top: 38px;
+    padding-bottom: 38px;
+  }
+
+  .nav-list {
+    display: block;
+    display: flex;
+    gap: 20px;
+  }
+
+  .nav-link {
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #1a2236;
+    font-family: var(--ff-normal);
+    text-decoration: none;
+  }
+
+  .hero-section {
+    padding-top: 0;
+    padding-inline-start: 90px;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .decoration {
+    display: none;
+  }
+
+  .decoration-desktop {
+    display: inline;
+    margin-right: 0;
+    flex: 1;
+  }
+
+  .hero-section1 {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    margin-bottom: 100px;
+  }
+
+  .hero-section1 .container {
+    flex: 1;
+  }
+
+  .bg-image {
+    background: url(./assets/desktop/image_dots.png) no-repeat;
+    background-position: right bottom;
+  }
+
+  .paragraph {
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 21px;
+    letter-spacing: -0.01em;
+    width: 500px;
+    height: 84px;
+    margin-top: 10px;
+  }
+
+  .cta-btn {
+    width: 254px;
+    height: 60px;
+    margin-bottom: 120px;
+  }
+
+  /* Project-section */
+  .project-header {
+    position: relative;
+    background: url(./assets/desktop/image_dots.png) no-repeat top right;
+  }
+
+  .rotate-one {
+    transform-origin: right top;
+    transform: rotate(-90deg);
+    position: absolute;
+    right: 90%;
+  }
+
+  .section {
+    padding-top: 0;
+    margin-bottom: 110px;
+  }
+
+  .grid {
+    display: grid;
+    grid-template-columns: 1fr 1fr 0.5fr 1fr;
+  }
+
+  .project-item {
+    position: relative;
+  }
+
+  .container-information {
+    position: absolute;
+    width: 100%;
+  }
```

Comment:
- [ ] Kindly make sure you use the correct font size and weigh in the projects section 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/188288381-c6ec0579-040f-4a64-957b-a4a981c388ca.png)  |  ![image](https://user-images.githubusercontent.com/74814780/188288368-8646ecef-9203-4e41-a7fb-986f373686ba.png)